src/main/java/mate/academy/hibernate/relations/model/Country.java

```diff
@@ -1,6 +1,16 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "countries")
 public class Country implements Cloneable {
```

Comment:
Implementing Cloneable and overriding the clone() method is unusual for JPA entities. It can lead to issues with entity identity and is generally not recommended unless there's a specific reason for it.